sortingHat/test_sortingHat.py

```diff
@@ -0,0 +1,48 @@
+import os
```

Comment:
Are we removing the testing directory? It's fine by me if we do, just want to make sure we're also consistent with the data directory existing or not (and, on that note it seems there's 2 test2014data.csv, one in sortingHat directory and one in test_data directory)